scripts/lab-control.py

```
@@ -100,7 +86,18 @@ def run_script(name):
         print(r(f"  Script not found: {path}"))
         print(y(f"  Make sure {name} is in the same directory as lab-control.py"))
         return
-    run(f"bash {path}")
+    print(dim(f"$ bash {path}"))
+    subprocess.run(["bash", path], check=False)
+
+
+def run_remote_script(ip, script):
+    """Send a script over stdin so sensitive values never enter argv."""
+    return subprocess.run(
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)